src/reflect/scala/reflect/internal/Symbols.scala

```diff
@@ -3433,7 +3433,7 @@ trait Symbols extends api.Symbols { self: SymbolTable =>
 
     override def derivedValueClassUnbox =
       // (info.decl(nme.unbox)) orElse      uncomment once we accept unbox methods
-      (info.decls.find(_ hasAllFlags PARAMACCESSOR | METHOD) getOrElse
+      (info.decls.reverseIterator.find(_ hasAllFlags PARAMACCESSOR | METHOD) getOrElse
```

Comment:
We might not see an improvement from `.decls.reverseIterator` if any other usages of this `decls` scope forces creation of the internally cached `List[Symbol]` (as opposed to `p.deferredMembers.iterator` above where we're working on a temporary `Scope`.)

But I'm still happy to accept this change.